### PR TITLE
isort to improve automation of code-review and code-consistency

### PR DESCRIPTION
**Pull request by ross-spencer**
Connected to #31

### CODE REVIEW

File: AIPscan/Aggregator/views.py [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 
 from AIPscan import db
-from AIPscan.Aggregator.task_helpers import get_packages_directory
-from AIPscan.Aggregator.forms import StorageServiceForm
 from AIPscan.Aggregator import tasks
+from AIPscan.Aggregator.forms import StorageServiceForm
+from AIPscan.Aggregator.task_helpers import get_packages_directory
 from AIPscan.extensions import celery
-from AIPscan.models import (
+from AIPscan.models import (  # Custom celery Models.
```
Comment (tw4l) [inline AIPscan/Aggregator/views.py:15]: This comment ended up in a funny place. Perhaps it could be removed?

File: tox.ini [hunk 2]
```diff
@@ -23,3 +23,11 @@ ignore =
...
 
 import-order-style = pep8
+
+[isort]
+multi_line_output = 3
+include_trailing_comma = True
+force_grid_wrap = 0
+use_parentheses = True
+ensure_newline_before_comments = True
+line_length = 88
```
Comment (tw4l) [inline tox.ini:33]: These are to get `isort` to work nicely with `black`, yeah? Nice attention to detail!

**Review by tw4l — APPROVED** [review]
Nice work, @ross-spencer! I left one comment where applying `isort` put a comment in an funny spot - otherwise, this looks great. It's a really nice addition to the project, and the approach of tying into a pre-commit hook is :man_cook: :ok_hand:

That only leaves one thing to say...

LGTM image

File: AIPscan/Aggregator/views.py (hunk 1 above)
Comment (ross-spencer) [reply AIPscan/Aggregator/views.py]: Oh weird, that may have started in a weird place too? No idea why! Correcting it seems to have put it back in a sensible place. Great catch!

**Before**

```python
from AIPscan.models import (	
    FetchJob,	# Custom celery Models.
    StorageService,	from AIPscan.models import FetchJob, StorageService, get_mets_tasks, package_tasks
    # Custom celery Models.	
    package_tasks,	
    get_mets_tasks,	
)
```

**After**

```python
# Custom celery Models.
from AIPscan.models import FetchJob, StorageService, get_mets_tasks, package_tasks
```

**Comment by ross-spencer** [comment]
Awesome @tw4l! Really glad to have this in here now. Thank you for the review/+1 on this.

**Pull request merged**